exporters/otlp/otlp.go

```diff
@@ -0,0 +1,266 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// code in this package is mostly copied from contrib.go.opencensus.io/exporter/ocagent/connection.go
+package otlp
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+	"unsafe"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+
+	coltracepb "github.com/open-telemetry/opentelemetry-proto/gen/go/collector/traces/v1"
+	tracepb "github.com/open-telemetry/opentelemetry-proto/gen/go/trace/v1"
+
+	export "go.opentelemetry.io/otel/sdk/export/trace"
+)
+
+var defaultTimeout = time.Duration(60 * time.Second)
+
+var _ export.SpanBatcher = (*Exporter)(nil)
+
+type Exporter struct {
+	// mu protects the non-atomic and non-channel variables
+	mu sync.RWMutex
+	// senderMu protects the concurrent unsafe send on traceExporter client
+	senderMu          sync.Mutex
+	started           bool
+	stopped           bool
+	traceExporter     coltracepb.TraceServiceClient
+	grpcClientConn    *grpc.ClientConn
+	lastConnectErrPtr unsafe.Pointer
```

Comment:
`lastConnectErrPtr` is used with atomics. Could you add the alignment test for this field?

Reply:
added.